### PR TITLE
Fixed issues with current price in graph, price changes (fixes #1204)

### LINKED ISSUE #1204: Some values become incorrect when clicking the date dropdown on the historic prices graph
This issue was originally discovered by @drptbl
-------------------------------------------------------------

**Wallet version or commit SHA:**
a91d1ae

**Environment:**
OS: Ubuntu 16.10
Arch: x86_64
Kernal Version: 4.11.2-041102-generic
(This also happens on Win 10 Home (64 bit) Version 1803 Build 17134.48, as reported by @drptbl )

**Prerequisites:**
- Active network connection

**Reproduction steps:**
e.g.:
1. Start Neon Wallet
2. Log into a saved wallet
3. Click the "1 month"/"1 day"/"1 week" button in the top right corner

some description (for ex. what happens after completing reproduction steps), e.g.:
- The current neo/gas value on the top center of historic price graph sometimes goes up or down by a cent
- The percentages in the "balances" section change to not reflect a single day of changes. The time span they represent decreases to an hour.

**Screenshots/Video:**
graph_issues

### PR DESCRIPTION
**Pull request by alexhulbert**
**What current issue(s) from Trello/Github does this address?**
Issue #1204 

**What problem does this PR solve?**
- The application would crash if you tried to successfully load the AssetBalances panel for the first time after loading the PriceHistory panel. This could happen if the wallet loaded without an internet connection
- The current price field on the PriceHistory panel changed by a cent or two sometimes on click of the "1 week"/"1 month"/"1 day" button
- The price change labels on the AssetBalances panel were inaccurate when the PriceHistory panel was set to "1 week" or "1 day" 

**How did you solve this problem?**
- I fixed crashes in the application by making sure that all prerequisite actions are loaded in both AssetBalances and PriceHistory.
- I fixed the changing price in the PriceHistory by adding a static current price value (the one used for fiat calculations in the AssetBalances panel) to the props
- I changed the percent changes in the AssetBalances so they always reflected the corresponding values in the top right corner.

**How did you make sure your solution works?**
- I clicked the asset and time buttons in the price history panel to make sure everything changed correctly
- I tried disconnecting and reconnecting the internet to see if all the correct actions were reloaded on click of the "retry" and "refresh" button

- [X] Unit tests written?

### CODE REVIEW

File: app/components/Dashboard/PriceHistoryPanel/PriceHistoryChart.jsx [hunk 1]
```diff
@@ -83,9 +84,10 @@ export default class PriceHistoryChart extends React.Component<Props> {
   }
 
   renderLatestPrice = () => {
+    let { staticPrice } = this.props
```
Comment (evgenyboxer) [inline app/components/Dashboard/PriceHistoryPanel/PriceHistoryChart.jsx]: Seems `const` should be used instead

File: app/components/Dashboard/PriceHistoryPanel/index.js [hunk 2]
```diff
@@ -1,34 +1,46 @@
...
+})
+
 const mapPriceHistoryActionsToProps = (actions, props) => ({
   setDuration: (duration: Duration) => {
     props.setDuration(duration)
     return actions.call({ duration, currency: props.currency })
   }
 })
 
+const batchActions = createBatchActions('priceHistoryPanel', {
```
Comment (evgenyboxer) [inline app/components/Dashboard/PriceHistoryPanel/index.js]: Could you move that into the `app/actions` directory?

Comment (alexhulbert) [reply app/components/Dashboard/PriceHistoryPanel/index.js]: I'll do this to all the batchActions instances (as they all contain a TODO asking to move them into a separate action file) and submit it as another PR.

**Comment by alexhulbert** [comment]
The build failed message can be safely ignored at this point. The error in Circle is irrelevant to the code in this PR. Another PR fixing the build issues has been submitted independently of this one.

**Comment by evgenyboxer** [comment]
Sure, np. Could you resolve this one conflict? and I'll merge the PR.
Awesome work.

**Pull request merged**